ansible_collections/arista/avd/roles/eos_l3ls_evpn/templates/evpn-fabric-l3leaf-yml.j2

```diff
@@ -253,6 +253,7 @@ bfd_multihop:
 {% include 'base/bfd-multihop.j2' %}
 
 {# Port-Channel Interfaces #}
+{% if leaf.mlag == true %}
```

Comment:
There are could some cases where `leaf.mlag == false` and there are still port-channels defined. For Example a port-channel from a server to a single-homed leaf. We should test for all use cases.

Reply:
Yes. In this case, need to add an additional complex test to define if PO is expected or not.

Reply:
Should be resolved with commit 2c610b9